src/executor/operator/physical_index_scan.cpp

```diff
@@ -32,21 +32,28 @@ import secondary_index_scan_execute_expression;
 import logical_type;
 import segment_column_index_entry;
 import segment_entry;
+import fast_rough_filter;
 // TODO:use bitset
 import bitmask;
 
 namespace infinity {
 
+void swap(infinity::Bitmask &left, infinity::Bitmask &right) {
+    left.Swap(right);
```

Comment:
1. remove Bitmask::Swap function.
2. Give a detailed comment on this function, why we need it?